src/interface/src/app/treatments/treatment-area-layer/treatment-area-layer.component.html

```diff
@@ -0,0 +1,21 @@
+<mgl-geojson-source [id]="sourceName">
+  <mgl-feature [geometry]="polygonGeometry"> </mgl-feature>
+</mgl-geojson-source>
+<mgl-layer
+  id="treatment-area-polygon"
+  type="line"
+  [source]="sourceName"
+  [paint]="{
+    'line-color': 'black',
+    'line-width': 2
+  }">
+</mgl-layer>
+<mgl-layer
+  id="treatment-area-fill"
+  type="fill"
+  [source]="sourceName"
+  [paint]="{
+    'fill-color': '#e9eded',
```

Comment:
Let me know if this color is ok, I picked it up from the area one using the devtools but I don't know if its clear enough